.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: required
 language: scala
 
 scala:
-  - 2.12.3
+  - 2.12.4
   - 2.11.11
```

Comment:
Since you're using `sbt +test` below, there isn't a need to declare versions to Travis. I'd just remove this scala section and simply let SBT take care of testing across versions.